pkg/gardenlet/controller/managedseed/managedseed_valueshelper_test.go

```diff
@@ -343,13 +352,43 @@ var _ = Describe("ValuesHelper", func() {
 		It("should compute the correct gardenlet chart values with bootstrap", func() {
 			result, err := vh.GetGardenletChartValues(mergedDeployment, mergedGardenletConfig(true), "bootstrap kubeconfig")
 			Expect(err).ToNot(HaveOccurred())
-			Expect(result).To(Equal(gardenletChartValues(true, "bootstrap kubeconfig")))
+			Expect(result).To(Equal(gardenletChartValues(true, "bootstrap kubeconfig", "node", 1, nil)))
 		})
 
 		It("should compute the correct gardenlet chart values without bootstrap", func() {
 			result, err := vh.GetGardenletChartValues(mergedDeployment, mergedGardenletConfig(false), "")
 			Expect(err).ToNot(HaveOccurred())
-			Expect(result).To(Equal(gardenletChartValues(false, "")))
+			Expect(result).To(Equal(gardenletChartValues(false, "", "node", 1, nil)))
+		})
+
+		It("should compute the correct gardenlet chart values if seed config is multi-zonal", func() {
```

Comment:
Should we add some more tests that check the following configurations: ReplicaCount > 1, FailureToleranceType = "node" (checks if "node" is preserved since it is already set); ReplicaCount=1, FailureToleranceType = nil (checks that `failureToleranceType` value is not added)?